src/components/Generic/Dialogs/UpdatePrinterDialog.vue

```diff
@@ -177,6 +180,17 @@ export default defineComponent({
       this.printersStore.setUpdateDialogPrinter(undefined);
       this.copyPasteConnectionString = "";
     },
+
+    async duplicatePrinter() {
+      if (!(await this.isValid())) return;
+      const formData = this.formData();
+      if (!formData) return;
+      const newPrinterData = PrintersService.convertCreateFormToPrinter(formData);
+      await this.printersStore.createPrinter(newPrinterData);
```

Comment:
You are correct, this will possibly leave behind orphaned printers because the user does not understand where it went. I will propose another way below👍🏼